packages/code-du-travail-frontend/src/search/__tests__/DocumentSuggester.test.js

```diff
@@ -98,6 +98,6 @@ describe("<DocumentSuggester />", () => {
     input.focus();
     const option = getByRole("option");
     option.click();
-    expect(onSelect).toHaveBeenCalledWith(suggestions[0]);
+    expect(onSelect).toHaveBeenCalledWith(suggestions[0], expect.anything());
```

Comment:
:thinking: That's in progress I guess

Reply:
for that one, it's because the second argument is  a synthetic event, but i've not found any exported contructor to use `expect.any(SyntheticEvent)`
I need to test it with Event perhaps